kolena/workflow/test_run.py

```diff
@@ -428,6 +457,28 @@ def _upload_test_sample_metrics(
         )
         krequests.raise_for_status(res)
 
+        if len(thresholded_metrics) > 0:
+            log.info("Uploading thresholded metrics")
+            df = pd.DataFrame(thresholded_metrics, columns=["test_sample", "metrics"])
+            df_validated = MetricsDataFrame(validate_df_schema(df, MetricsDataFrameSchema, trusted=True))
+            df_serializable = df_validated.as_serializable()
+            log.info(df_serializable)
```

Comment:
I assume this should be removed? We wouldn't want to log an entire data frame.

Reply:
removed